jdk17-image/image.yaml

```diff
@@ -0,0 +1,162 @@
+schema_version: 1
+
+name: "jboss-eap-7-tech-preview/eap74-openjdk17-openshift-rhel8"
+description: "Red Hat JBoss Enterprise Application Platform 7.4 OpenShift container image."
+version: "7.4.5"
+from: "registry.redhat.io/ubi8/ubi:latest"
+labels:
+    - name: "com.redhat.component"
+      value: "jboss-eap-74-openjdk17-builder-openshift-rhel8-container"
+    - name: "io.k8s.description"
+      value: "Platform for building and running JavaEE applications on JBoss EAP 7.4"
+    - name: "io.k8s.display-name"
+      value: "JBoss EAP 7.4"
+    - name: "io.openshift.expose-services"
+      value: "8080:http"
+    - name: "io.openshift.tags"
+      value: "builder,javaee,eap,eap7"
+    - name: "io.openshift.s2i.scripts-url"
+      value: "image:///usr/local/s2i"
+    - name: "maintainer"
+      value: "Red Hat"
+envs:
+      - name: DISABLE_LEGACY_SECURITY
+        value: "true"
+      - name: AB_JOLOKIA_OFF
```

Comment:
It seems from discussions that we are not disabling Jolokia for this release.

Reply:
+1 updated